easybuild/easyconfigs/o/OpenMPI/OpenMPI-2.0.1-iccifort-2017.1.132-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '2.0.1'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'iccifort', 'version': '2017.1.132-GCC-5.4.0-2.26'}
+
+sources = [SOURCELOWER_TAR_GZ]
+
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads',]
+
+dependencies = [('hwloc', '1.11.4')]
+
+configopts = '--with-threads=posix --enable-shared --enable-mpi-thread-multiple --with-verbs'
```

Comment:
@akesandgren you're missing a space after `--with-verbs` here, which is going to lead to `--with-verbs--enable-mpirun-prefix-by-default` rather than `--with-verbs --enable-mpirun-prefix-by-default`...